projects/igniteui-angular/src/lib/input-group/input-group.component.spec.ts

```diff
@@ -178,26 +178,50 @@ describe('IgxInputGroup', () => {
         expect(inputGroupElement.classList.contains(INPUT_GROUP_COMPACT_DENSITY_CSS_CLASS)).toBeTruthy();
     });
 
-    it('suppress focus on input when clicked', async () => {
-        const fixture = TestBed.createComponent(InputGroupSupressInputFocusComponent);
```

Comment:
If there's no other test using this options.. might be a good idea to keep it

Reply:
new test added to test this